minigrid/__init__.py

```diff
@@ -597,7 +597,7 @@ def register_minigrid_envs():
     register(
         id="BabyAI-GoToObjS6-v0",
         entry_point="minigrid.envs.babyai:GoToObj",
-        kwargs={"room_size": 4},
+        kwargs={"room_size": 6},
```

Comment:
@BolunDai0216 Should we change the version number as well?

Reply:
If the room size is the only change then I think we should keep the version number.

Reply:
But to users this is modify how agents will act

Reply:
@mansicer Could you change the version number of the environment in `minigrid/__init__.py` to v1
Also could you add a note in the environment docstring for the reason that the version number change along with version

Reply:
@pseudo-rnd-thoughts I have made corresponding changes. Should we keep the v0 registry available rather than removing it?

Reply:
No, as it is equivalent to `BabyAI-GoToObjS4-v0`